src/conversions.jl

```diff
@@ -284,6 +284,13 @@ function el32convert(x::AbstractArray{T, N}) where {T<:Union{Missing, <: Number}
     end::Array{Float32, N}
 end
 
+"""
+Takes an input `LineString` and decomposes it to points.
+"""
+function convert_arguments(PB::PointBased, linestring::LineString)
+    return convert_arguments(PB, linestring.points.parent.data)
```

Comment:
How about using `decompose(Point, linestring)` ?

Reply:
So that gives an error, 
```julia
ls1=LineString([Point(1, 2), Point(4,5), Point(10, 8), Point(1, 2)])

decompose(Point, decompose(GB.Ngon, ls1))
ERROR: MethodError: no method matching coordinates(::Array{GeometryBasics.Ngon,1})
Closest candidates are:
  coordinates(::GeometryBasics.Ngon) at /home/ezio/.julia/packages/GeometryBasics/CMxB4/src/basic_types.jl:76
  coordinates(::Simplex) at /home/ezio/.julia/packages/GeometryBasics/CMxB4/src/basic_types.jl:146
  coordinates(::LineString) at /home/ezio/.julia/packages/GeometryBasics/CMxB4/src/basic_types.jl:184
  ...
Stacktrace:
 [1] decompose(::Type{Point}, ::Array{GeometryBasics.Ngon,1}) at /home/ezio/.julia/packages/GeometryBasics/CMxB4/src/interfaces.jl:104
 [2] top-level scope at REPL[59]:1
```

So this is quite synonymous to https://github.com/JuliaGeometry/GeometryBasics.jl/issues/69

Reply:
Also the `decompose` method seems to be slow, and in our case we will need some nested decomposition maybe.

Reply:
https://github.com/JuliaGeometry/GeometryBasics.jl/pull/73